packages/syft/src/syft/service/network/network_service.py

```diff
@@ -464,6 +448,367 @@ def add_veilid_peer(
 
         return self_node_peer
 
+    @service_method(path="network.add_route_for", name="add_route_for")
+    def add_route_for(
+        self,
+        context: AuthedServiceContext,
+        peer: NodePeer,
```

Comment:
suggestion: rename this method to `add_route_on_peer` ?
Update docstring to 

`Add or update the route information on the remote peer.`